### ISSUE TITLE
Clarify Logo Settings & Shortcut Icon Settings wording at /admin/config/system/site-information

### ISSUE THREAD

**Issue by ColumbiaBlooms** [backdrop/backdrop-issues#2601]
The wording and functionality of Logo Settings and Shortcut Icon Settings on the Site Information page (/admin/config/system/site-information) is unclear.

Please clearly define what Logo Settings are and the relationship between uploading a logo (Choose File button) and the Path to custom logo box. It would seem that uploading a logo would automatically set the Path? Why is Path needed?

Above comments apply to Shortcut Icon Settings.

**Comment by olafgrabienski** [comment]
Yes, I find it also unclear. When I remember correct (from Drupal theme settings which seam to be the same), the path is set automatically when you upload a logo or favicon. That's ok in my opinion. But as for the other option, I never know where to upload a file and/or which path to fill in manually. Can someone explain the relation between custom paths and upload destinations?
